tests/access/test_Ownable.py

```diff
@@ -85,6 +86,12 @@ async def test_renounceOwnership(ownable_factory):
     executed_info = await ownable.owner().call()
     assert executed_info.result == (ZERO_ADDRESS,)
 
+    # Protected function cannot be called from zero address
+    await assert_revert(
+        ownable.protected_function().call(),
+        reverted_with="Ownable: caller is not the owner"
+    )
+
```

Comment:
I don't think this belongs to this test. This assert is not testing the `renounceOwnership` function, but `assert_only_owner`. Put differently if this assert fails, one should not think there's a problem with `renounceOwnership`.